dev-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3 meson
+
+DESCRIPTION="A modular Wayland compositor library"
+HOMEPAGE="https://github.com/swaywm/wlroots"
+
+EGIT_REPO_URI="https://github.com/swaywm/wlroots"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE="caps elogind systemd xcb-errors xwayland X"
+
+DEPEND="
+>=dev-libs/wayland-protocols-1.12
+>=media-libs/mesa-17.1.0[egl,gles2,gbm]
+dev-libs/libinput
+dev-libs/wayland
+virtual/libudev
+x11-libs/libdrm
+x11-libs/libxkbcommon
+x11-libs/pixman
+caps? ( sys-libs/libcap )
+elogind? ( sys-auth/elogind )
+systemd? ( sys-apps/systemd )
+xwayland? (
+	x11-base/xorg-server[wayland]
+	x11-libs/libxcb
+	x11-proto/xcb-proto
+)
+X? (
```

Comment:
There are duplicated dependencies between `X` and `xwayland` but I couldn't find a way to not repeat them.

Reply:
It's OK to duplicate them. For example weston ebuild does it.

Reply:
I'll add it.